legal-api/tests/unit/services/warnings/test_warnings.py

```diff
@@ -27,33 +27,44 @@
 
 
 @pytest.mark.parametrize(
-    'test_name, legal_type, identifier, has_office, num_persons_roles, num_org_roles, filing_types, filing_has_completing_party, expected_code, expected_msg',
+    'test_name, legal_type, identifier, has_office, num_persons_roles, num_org_roles, filing_types, filing_has_completing_party, cessation_date, expected_code, expected_msg',
     [
         # SP tests
-        ('SUCCESS', 'SP', 'FM0000001', True, 1, 0, ['registration'], [True], None, None),
-        ('SUCCESS', 'SP', 'FM0000001', True, 0, 1, ['registration'], [True], None, None),
-        ('SUCCESS', 'SP', 'FM0000001', True, 1, 0, ['registration', 'conversion'], [False, True], None, None),
-        ('SUCCESS', 'SP', 'FM0000001', True, 0, 1, ['registration', 'conversion'], [False, True], None, None),
-        ('FAIL_NO_PROPRIETOR', 'SP', 'FM0000001', True, 0, 0, ['registration'], [True], 'NO_PROPRIETOR', 'A proprietor is required.'),
-        ('FAIL_NO_OFFICE', 'SP', 'FM0000001', False, 1, 0, ['registration'], [True], 'NO_BUSINESS_OFFICE', 'A business office is required.'),
-        ('FAIL_NO_COMPLETING_PARTY', 'SP', 'FM0000001', True, 1, 0, ['registration'], [False], 'NO_COMPLETING_PARTY', 'A completing party is required.'),
+        ('SUCCESS', 'SP', 'FM0000001', True, 1, 0, ['registration'], [True], None, None, None),
+        ('SUCCESS', 'SP', 'FM0000001', True, 0, 1, ['registration'], [True], None, None, None),
+        ('SUCCESS', 'SP', 'FM0000001', True, 1, 0, ['registration', 'conversion'], [False, True], None, None, None),
+        ('SUCCESS', 'SP', 'FM0000001', True, 0, 1, ['registration', 'conversion'], [False, True], None, None, None),
+        ('FAIL_NO_PROPRIETOR', 'SP', 'FM0000001', True, 0, 0, ['registration'], [True], None, 'NO_PROPRIETOR', 'A proprietor is required.'),
+        ('FAIL_NO_OFFICE', 'SP', 'FM0000001', False, 1, 0, ['registration'], [True], None, 'NO_BUSINESS_OFFICE', 'A business office is required.'),
+        ('FAIL_NO_COMPLETING_PARTY', 'SP', 'FM0000001', True, 1, 0, ['registration'], [False], None, 'NO_COMPLETING_PARTY', 'A completing party is required.'),
+        ('SUCCESS', 'SP', 'FM0000001', True, 1, 0, ['registration'], [True], datetime.utcnow(), 'NO_PROPRIETOR', 'A proprietor is required.'),
+        ('SUCCESS', 'SP', 'FM0000001', True, 0, 1, ['registration'], [True], datetime.utcnow(), 'NO_PROPRIETOR', 'A proprietor is required.'),
+        ('SUCCESS', 'SP', 'FM0000001', True, 1, 0, ['registration', 'conversion'], [False, True], datetime.utcnow(), 'NO_PROPRIETOR', 'A proprietor is required.'),
+        ('SUCCESS', 'SP', 'FM0000001', True, 0, 1, ['registration', 'conversion'], [False, True], datetime.utcnow(), 'NO_PROPRIETOR', 'A proprietor is required.'),
```

Comment:
can we rename these tests?  maybe something like `FAIL_NO_PROPRIETOR_CESSATION_DATE`

Reply:
Oops. I forgot to change them as I did for partners.

Reply:
Oops. I forgot to change them as I did for partners.